app/middlewares/cartMiddleware.js

```diff
@@ -27,4 +29,15 @@ const validateCartRequestBody = (req, res, next) => {
   next()
 }
 
-export { validateCartRequestBody }
+const validateCartRequestQuantityBody = (req, res, next) => {
```

Comment:
ini functionnya dipake dimana van?
nek tak baca fungsine sama aja kyk validateCartRequestBody

jd mending ga usah, pas update qty item in cart kan, request body nya sama ky add item to cart kan?